src/App.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <div class="app-container" v-bind:class="collapseSidebar ? 'sidebar-collapse': 'sidebar-expand'">
+  <div class="app-container" v-bind:class="collapseSidebar ? 'sidebar-toggle': 'null'">
```

Comment:
So that you are not adding a class called 'null' on your app container, you can also use this syntax for conditionally adding classes:

`:class="{'sidebar-toggle': collapseSidebar}`

Docs: https://v3.vuejs.org/guide/class-and-style.html

Reply:
Good catch, that was supposed to be null not 'null'! The Vue syntax is even better, I've updated it :)